changelogs/fragments/10178-complete-fix-of-typeerror-cobbler-xmlrpc.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - cobbler_system - fix bug with Cobbler >= 3.4.0 caused by giving more than 2 positional arguments to ``CobblerXMLRPCInterface.get_system_handle()`` (https://github.com/ansible-collections/community.general/issues/8506, https://github.com/ansible-collections/community.general/pull/10145, https://github.com/ansible-collections/community.general/pull/10178).
```

Comment:
Please create a new changelog fragment file. Modifying fragments that have alrady been included in releases is generally not OK. (This has been in 11.7.0: https://github.com/ansible-collections/community.general/blob/stable-10/CHANGELOG.md#bugfixes.)